### ISSUE TITLE
Print list of files with unknown extension

### ISSUE THREAD

**Issue by nochso** [DiSiqueira/Gorganizer#13]
This way it'd be really easy too see what kind of rules you might be missing.

**Comment by d6o** [comment]
Do.you.think this should be a parameter flag or a list printed after moving files?

**Comment by nochso** [comment]
Just a list. Maybe combine it with the output of issue #12 ? For example

```
Videos
    a.mkv

Unknown extension (will not be moved)
    a.xxx
    foo.bar
```

**Comment by d6o** [comment]
Sorry about this delay, implemented in #22
